src/Workspaces/CoreTestUtilities/WorkspaceTestModuleInitializer.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+using Microsoft.CodeAnalysis.ErrorReporting;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.UnitTests
+{
+    internal static class WorkspaceTestModuleInitializer
+    {
+        [ModuleInitializer]
+        public static void ModuleInitializer()
```

Comment:
❔ Why is it not suitable to just use the one above?

Reply:
Which one? The one in compiler tests? We have two sets of handlers - one for compiler layer one for workspace layer.